packages/ui/ButtonGroup/index.tsx

```diff
@@ -137,3 +122,24 @@ const ButtonGroupStyled = styled.div<GroupProps>`
           `}
   }
 `;
+
+export const ButtonGroup = ({ options, secondary }: GroupProps) => {
+  return (
+    <ButtonGroupStyled options={options} secondary={secondary}>
+      {options.map(({ component, onSelect, defaultChecked, id }, index) => {
+        return (
+          <Fragment key={id}>
+            <input
+              type={"radio"}
+              id={`button-${index}`}
```

Comment:
`id` should not be based on `index`. If there were two button groups on one page, then in both of them there would be a button with for example `id="button-0"`.
```suggestion
              id={`button-${id}`}
```

Reply:
same as: https://github.com/intive/patronage2023-js/pull/100#discussion_r1197623477

Reply:
I know,  just caught my eye :) would be good to improve it anyway if you have a moment 🤗

Reply:
done